### PR TITLE
Update repositories to latests Crystal release

### PR DESCRIPTION
**Pull request by BorjaOuterelo**
This updates repository lists to the latest Crystal release.
Reorganise repositories lists to make the difference with future releases easier.

### CODE REVIEW

File: Installation/repos/agent_minimum.repos [hunk 1]
```diff
@@ -102,34 +100,38 @@ repositories:
...
-    type: git
-    url: https://github.com/microROS/rclc.git
-    # Change2Tag
-    version: feature/Crystal_update
   uros/micro-ROS-demo:
     type: git
     url: https://github.com/microROS/micro-ROS-demos.git
-    # Change2Tag
-    version: develop
+    version: develop
```
Comment (iluetkeb) [inline Installation/repos/agent_minimum.repos]: Maybe we can also add a tag like release-crystal-20190312, as above.

File: Installation/repos/agent_minimum.repos [hunk 2]
```diff
@@ -102,34 +100,38 @@ repositories:
...
+
+# Changes from ROS 2
+  uros/rclc:
+    type: git
+    url: https://github.com/microROS/rclc.git
+    version: release-crystal-20190312
+  uros/osrf_testing_tools_cpp:
+    type: git
+    url: https://github.com/microROS/osrf_testing_tools_cpp.git
+    version: v0.0.1-alpha
```
Comment (iluetkeb) [inline Installation/repos/agent_minimum.repos]: For the upstream version, we use 1.7.0 -- why can't we use the same here?

File: Installation/repos/agent_minimum.repos [hunk 3]
```diff
@@ -102,34 +100,38 @@ repositories:
...
 
 # Micro XRCE-DDS
   eProsima/Micro-XRCE-DDS-Agent:
     type: git
     url: https://github.com/eProsima/Micro-XRCE-DDS-Agent.git
-    # Change2Tag
-    version: master
+    version: master
```
Comment (iluetkeb) [inline Installation/repos/agent_minimum.repos]: Better make a tag as well, or we might break people's builds on updates.

File: Installation/repos/agent_minimum.repos (hunk 1 above)
Comment (BorjaOuterelo) [reply Installation/repos/agent_minimum.repos]: Done.

File: Installation/repos/agent_minimum.repos (hunk 2 above)
Comment (BorjaOuterelo) [reply Installation/repos/agent_minimum.repos]: 1.7.0 of osrf_testing_tools_cpp?. We use a modified version of osrf_testing_tools_cpp containing some fix for visual studio.

File: Installation/repos/agent_minimum.repos (hunk 3 above)
Comment (BorjaOuterelo) [reply Installation/repos/agent_minimum.repos]: Change to use the latest released version.

File: Installation/repos/agent_minimum.repos (hunk 2 above)
Comment (iluetkeb) [reply Installation/repos/agent_minimum.repos]: ok for now, but could we submit that fix to upstream as well? That would be preferable, IMHO.

Comment (BorjaOuterelo) [reply Installation/repos/agent_minimum.repos]: I added an issue in (https://github.com/microROS/osrf_testing_tools_cpp/issues/2) our fork, so we align with upstream soon.

**Pull request merged**